From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pancakse <pancakseyt@gmail.com>
Date: Sun, 5 Nov 2023 16:59:07 +0800
Subject: [PATCH] Netherite Server Config


diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index 8d8fe04e6b09d2a5b1cc05002073df5c58cdcb96..09fa5d4f8cbaf77150459f78d08004f85dd3cdcc 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -2120,6 +2120,14 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
             throw new UnsupportedOperationException("Not supported yet.");
         }
         // Paper end
+		
+		// Netherite start - Not supported yet too
+		@NotNull
+		public org.bukkit.configuration.file.YamlConfiguration getNetheriteConfig()
+        {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+		// Netherite end - Not supported yet too
 
         /**
          * Sends the component to the player
